MqttOutput.go

```diff
@@ -0,0 +1,70 @@
+package dnstap
+
+import (
+	mqtt "github.com/eclipse/paho.mqtt.golang"
+	"google.golang.org/protobuf/proto"
+)
+
+// MqttOutput implements a dnstap Output that publishes dnstap data to an MQTT broker.
+type MqttOutput struct {
+	outputChannel chan []byte
+	client        mqtt.Client
+	baseTopic     string
+	qos           byte
+	wait          chan bool
+	log           Logger
+}
+
+// NewMqttOutput creates a MqttOutput for publishing dnstap data to an MQTT broker.
+func NewMqttOutput(opts *mqtt.ClientOptions, baseTopic string, qos byte) (o *MqttOutput, err error) {
+	o = new(MqttOutput)
+	o.baseTopic = baseTopic
+	o.qos = qos
+	o.outputChannel = make(chan []byte, outputChannelSize)
+	o.wait = make(chan bool)
+	client := mqtt.NewClient(opts)
```

Comment:
```suggestion
	o := &MqttOutput{
	  baseTopic: baseTopic,
	  qos: qos,
	  outputChannel: make(chan []byte, outputChannelSize),
	  wait: make(chan bool),
	  client: mqtt.NewClient(opts),
	}
```
To be consistent. Then adjust indentation.